packages/custom-esbuild/src/load-modules.ts

```diff
@@ -0,0 +1,25 @@
+import { Path, getSystemPath, logging } from '@angular-devkit/core';
```

Comment:
I wonder if we should take this code into a common package (it becomes a pattern already, used in [`custom-webpack`](https://github.com/just-jeb/angular-builders/blob/master/packages/custom-webpack/src/utils.ts), [`jest`](https://github.com/just-jeb/angular-builders/blob/master/packages/jest/src/utils.ts), here). WDYT? Looks like a good opportunity for a little refactoring.

Reply:
If we decide to go with a `angular-builders/common`, then [`merge-schemes.ts`](https://github.com/just-jeb/angular-builders/blob/master/merge-schemes.ts) from the root can also go there.